v0.3/cruorg/components/input/input.scss

```diff
@@ -0,0 +1,139 @@
+        width: $cru-input-width;
+        &::placeholder {
+            color: $cru-input-placeholder-color;
+        }
+        &:focus {
+            outline: $cru-input-focus-outline;
+            border: $cru-input-focus-border;
+            box-shadow: $cru-input-focus-box-shadow;
+            caret-color: $cru-input-focus-caret-color;
+        }
+        &:focus::placeholder {
+            color: transparent;
+        }
+        &:disabled {
+            border-width: $cru-input-disabled-border-width;
+            border-style: $cru-input-disabled-border-style;
+            border-color: $cru-input-disabled-border-color;
+            color: $cru-input-disabled-color;
+            font-weight: $cru-input-disabled-font-weight;
+        }
+        &:invalid {
+            border-color: $cru-input-invalid-border-color;
+        }
+    }
+}
+
+@mixin cru-form-input-without-borders {
+    &__text {
+        border-top: none;
+        border-left: none;
+        border-right: none;
+        border-radius: 0;
+        &:focus {
+            box-shadow: none;
+            border-bottom: $cru-input-border-width $cru-input-border-style $cru-color-yellow;
+        }
+    }
+}
+
+@mixin cru-form-input-black {
+    label, span {
+        color: $cru-color-white;
+    }
+    &__text {
+        background-color: $cru-color-gray-dark;
+        border-color: $cru-color-white;
+        color: $cru-color-white;
+        &::placeholder {
+            color: $cru-color-white;
+        }
+        &:focus {
+            background-color: $cru-color-white;
+            color: $cru-color-gray-dark;
+        }
+        &:disabled, &:disabled::placeholder {
+            border-color: $cru-color-gray-medium;
+            color: $cru-color-gray-medium;
+        }
+    }
+}
+
+@mixin cru-form-input-black-without-borders {
+    @include cru-form-input-without-borders;
+    @include cru-form-input-black;
+    &__text{
+        &:focus {
+            background-color: $cru-color-gray-dark;
+            color: $cru-color-white;
+        }
+        &:focus::placeholder {
+            color: transparent;
+        }
+    }
+}
+
+.cmp-form {
+    .text {
+        @include cru-text;
+        margin: $cru-spacer-base;
+        width: 352px;
+        position: relative;
+        .cmp-form-text {
+            label, span {
+                @include cru-text-sm;
+                display: block;
+            }
+            svg {
+                position: absolute;
+                right: 4px;
+                bottom: 34px;
+            }
+            @include cru-form-input;
+        }
+        &.cru-form-text-without-borders .cmp-form-text {
+            @include cru-form-input-without-borders;
+        }
+        &.cru-form-text-black .cmp-form-text {
+            @include cru-form-input-black;
+        }
+        &.cru-form-text-black-without-borders .cmp-form-text {
+            @include cru-form-input-black-without-borders;
+        }
+    }
+}
```

Comment:
@sebamelzi More single-use mixins.

Reply:
@sebamelzi The height of the text fields is too short. They should be 48px tall and I'm only seeing 33px. And that height shrinks down to 31px when you click on it. I suspect the fields are losing their borders on focus.

Reply:
@sebamelzi  The invalid field borders are losing their yellow color on dark backgrounds.

Reply:
@sebamelzi The "without-borders" fields shouldn't have side padding.

Reply:
@sebamelzi The font size in the fields appears to be using the browser default but it should be 16px.

Reply:
> @sebamelzi The height of the text fields is too short. They should be 48px tall and I'm only seeing 33px. And that height shrinks down to 31px when you click on it. I suspect the fields are losing their borders on focus.

No, actually I did it on purpose because the active input field on figma does not have border.